File: 1. Two Sum - Easy.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}
        for idx, i in enumerate(nums):
            if (target - i) in hashMap:
                return [idx, hashMap[target - i]]
            hashMap[i] = hashMap.get(nums[idx], idx) 

sol = Solution()
print(sol.twoSum([2,7,11,15], 9))

File: 1. Two Sum - Easy.ts
function twoSum(nums: number[], target: number):{
  let check: string = "";
  for (let index = 0; index < nums.length; index++) {
    if (check.length !== 0) {
      break;
    }
    for (let i = 0; i < nums.length; i++) {
      if (i !== index) {
        if (nums[index] + nums[i] === target) {
          check = "found";
          return [index, i];
        }
      }
    }
  }
}
